Drop Debug fields

gibizer opened the pull request:
These fields were redundant. Crashing pods can be debugged via oc debug.

Implements: [OSPRH-4290](https://issues.redhat.com//browse/OSPRH-4290)

gibizer commented:
@kajinamit I rebased this PR to the webhook work that moved the default ContainerImage to an env variable. Could you please check that this look OK from your perspective?

SeanMooney commented:
should we just swap to antelope at this point?
we will need to do that sonner rater then later.

the images already exist and i would prefer to avoid changing these twice in a few weeks
https://quay.io/organization/podified-antelope-centos9

kajinamit commented:
> @kajinamit I rebased this PR to the webhook work that moved the default ContainerImage to an env variable. Could you please check that this look OK from your perspective?

Thanks. It looks good to me. I didn't push these replacement patches because kuttl tests fail unless we remove hard-code in install_yamls but it seems placement-operator is not affected because kuttl tests are not yet added so we are good.

kajinamit commented:
> should we just swap to antelope at this point? we will need to do that sonner rater then later.
> 
> the images already exist and i would prefer to avoid changing these twice in a few weeks https://quay.io/organization/podified-antelope-centos9

I think we can skip zed and wait for antelope. afaik rdo antelope is still being prepared so we probably have wait for another week to get the new images built with antelope packages but I'll double check it.

gibizer commented:
I'm OK to wait for the proper antelope packages.

kajinamit commented:
I confirmed these podified-antelope-centos9 images are built from RDO antelope packages so we can use these now.

gibizer commented:
I was able to run the antelope placement now:

```
$ openstack trait list | grep COMPUTE_SHARE_LOCAL_FS
| COMPUTE_SHARE_LOCAL_FS                |
```

gibizer reviewed (approved):
I think this is good to go

gibizer committed:
Drop Debug fields

These fields were redundant. Crashing pods can be debugged via oc debug.

Implements: OSPRH-4290

gibizer commented:
missed the kuttl asserts, fixed now.

stuggi reviewed (approved):
/lgtm
